src/Phansible/Roles/Xdebug.php

```diff
@@ -34,4 +36,17 @@ public function getInitialValues()
           'settings' => [],
         ];
     }
+
+    public function transformValues(array $values, VagrantBundle $vagrantBundle)
+    {
+        if ($values['install'] == 1) {
+            $values['settings'] = [
```

Comment:
This function was more conceived to do this: https://github.com/phansible/phansible/blob/db961b2c79883eb43e73852d5d552aba483b3364/src/Phansible/Roles/Php.php#L53

We only had 2 special cases. And I'm not really happy about having this function... so I don't want to encourage the usage of it.

I understand that you pretty much no one will want to change this settings.

The other way to resolve this, and maybe a better approach. Will be to set this options in the default folder.

This way you could add to `all.yml` the options and override the default ones.